commune_app/templates/commune_app/login.html

```diff
@@ -13,7 +13,7 @@
                 <form method="post">{% csrf_token %}
 
                     <div class="d-flex flex-row align-items-center mb-1">
-                        <input type="email" placeholder="Email address *" class="form-control m-1" name="email" />
+                        <input type="text" placeholder="Username *" class="form-control m-1" name="username" />
```

Comment:
Please look at:
https://github.com/redhat-beyond/commune/blob/main/commune_app/templates/commune_app/login.html#L16
These changes are already in the code.
Could you please rebase this PR on the latest code?

Reply:
I tried rebasing again @kobihk